provd/proto/theme.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+package theme;
+
+service ThemeService {
+    rpc SetBrightness(SetBrightnessRequest) returns (Empty);
+    rpc GetBrightness(Empty) returns (GetBrightnessResponse);
```

Comment:
I think the light/dark theme is call 'variant'? Brightness is too similar to screen brightness imo.